packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/NotebookSessionController.java

```diff
@@ -255,7 +254,12 @@ ResponseEntity<NotebookSession> cloneNotebookSession(
 			if (session.isEmpty()) {
 				return ResponseEntity.notFound().build();
 			}
-			return ResponseEntity.status(HttpStatus.OK).body(session.get().clone());
+			final NotebookSession newNotebookSession = sessionService.createAsset(
+				session.get().clone(),
+				projectId,
+				permission
+			);
+			return ResponseEntity.status(HttpStatus.OK).body(newNotebookSession);
```

Comment:
laksdjflksdajflksadjf

Reply:
You're right, this is wasn't being adding to the project asset table when cloning. I've added some additional lines to to this and it is now being slotted into the project asst table